libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/HttpHelper.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Microsoft.Bot.Builder.Integration.AspNet.Core
 {
-    internal static class HttpHelper
+    public static class HttpHelper
```

Comment:
If we make it public, we should make this helper more generic, for example, instead of public static async Task<Activity> ReadRequestAsync(HttpRequest request), we should expose public static async Task<T> ReadRequestAsync(HttpRequest request) so we can use it to read other types from the request body (like ConversationParameters or other things that may come in the body)